WebRoot/messages/ZMsg_iw.properties

```diff
@@ -39,18 +39,18 @@ account.PASSWORD_LOCKED = \u05d4\u05e1\u05d9\u05e1\u05de\u05d4 \u05e0\u05e2\u05d
 account.PASSWORD_RECENTLY_USED = \u05d4\u05e1\u05d9\u05e1\u05de\u05d4 \u05e9\u05e9\u05dc\u05d7\u05ea \u05d4\u05d9\u05d9\u05ea\u05d4 \u05d1\u05e9\u05d9\u05de\u05d5\u05e9 \u05dc\u05d0\u05d7\u05e8\u05d5\u05e0\u05d4. \u05d1\u05d7\u05e8 \u05e1\u05d9\u05e1\u05de\u05d4 \u05d0\u05d7\u05e8\u05ea.
 
 # {0} = filename
-formatter.INVALID_FORMAT = \u05ea\u05d1\u05e0\u05d9\u05ea \u05e7\u05d5\u05d1\u05e5 \u05dc\u05d0-\u05d7\u05d5\u05e7\u05d9\u05ea \u05e2\u05d1\u05d5\u05e8 \u05d4\u05e7\u05d5\u05d1\u05e5 '{0}'.
+formatter.INVALID_FORMAT = \u05ea\u05d1\u05e0\u05d9\u05ea \u05e7\u05d5\u05d1\u05e5 \u05dc\u05d0-\u05d7\u05d5\u05e7\u05d9\u05ea \u05e2\u05d1\u05d5\u05e8 \u05d4\u05e7\u05d5\u05d1\u05e5 ''{0}''.

```

Comment:
@zx-fulvio

Totally agree with what you said above. 
I was going to get back to you on this one, because I recently pulled all the repos for some general checks I wanted to make and looked for this apostrophe issue as well.

My initial impression was that most apostrophes were doubled where needed, and Catalan was the one exception where this was treated differently. It's not and indeed there's a lot of fixing needed in all files/languages.

I think method (II) is the absolute best in this case.
For our other clients that use this file format, the smart quote is the preferred method and for the languages we have in common with them I know a smart quote is accepted. I will have to confirm this for languages like Hebrew, where this might be an issue, but I can fix this with either bidi marks or simply doubling the quotes if I have no other choice.
for instances such as 'a@example.com' I'm thinking of using opening and closing single quotes, if you agree.

I have a semi-automated method of doing this globally and it works decently. I've done it for Catalan and it went well.
I can do this in all languages (yours included and add you for review if you want).